docs/source/getting-started/installation.rst

```diff
@@ -544,11 +544,16 @@ Nebius
   nebius iam get-access-token > ~/.nebius/NEBIUS_IAM_TOKEN.txt
   nebius --format json iam whoami|jq -r '.user_profile.tenants[0].tenant_id' > ~/.nebius/NEBIUS_TENANT_ID.txt
 
-**Optional**: You can specify a preferable project ID, which will be used if a project ID is required in the designated region.
 
-.. code-block:: shell
+**Optional**: You can specify a preferable project ID and fabric in `~/.sky/config.yaml`
```

Comment:
Question about "preferable" - does this mean that the project id / fabric can be ignored in some cases, or is it always respected?